app/models/patient_session.rb

```diff
@@ -99,11 +108,16 @@ def able_to_vaccinate?
   def safe_to_destroy?
     vaccination_records.empty? && gillick_assessments.empty?
   end
+  alias_method :safe_to_transfer?, :safe_to_destroy?
 
   def destroy_if_safe!
     destroy! if safe_to_destroy?
   end
 
+  def propose_transfer_if_safe!(session)
+    update!(proposed_session: session) if safe_to_transfer?
```

Comment:
I think we should always set a `proposed_session`, but when we come to build the UI to confirm the move, only if it's `safe_to_transfer?` do we _remove_ the patient from their existing session while also adding them to the proposed session.

Reply:
Yeah, this makes sense!

Reply:
Will be handled in the UI in a follow-up